optimize/client/src/modules/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -0,0 +1,3 @@
+.Breadcrumbs {
+  margin: 0 40px;
```

Comment:
I could not do this because the breadcrumbs needs to be alighted with the rest of the layout which does not have carbon spacing, but thanks for raising the comment. I created https://github.com/camunda/camunda/issues/24716
to address this for the whole layout.